src/python/ensembl/io/genomio/manifest/check_integrity.py

```diff
@@ -78,22 +78,32 @@ def __init__(self, manifest_path: PathLike) -> None:
         self.brc_mode = False
 
     def has_lengths(self, name: str) -> bool:
-        """Check if a given name has lengths records."""
-        if name in self.lengths:
-            return True
-        return False
+        """Check if a given name has lengths records.
+        
+        Raise KeyError if the name is not supported.
+
+        """
+        try:
+            if self.lengths[name]:
+                return True
+            return False
```

Comment:
Just to double-check: if `self.lengths[name]` exists but the dictionary inside is empty, `False` will be returned as well. This was the behaviour before, but want to confirm that is what we want.

Reply:
That's correct